src/game/overlay.c

```diff
@@ -172,7 +172,8 @@ static void Overlay_DrawBar(int32_t value, int32_t value_max, int32_t bar_type)
     int32_t blink =
         percent <= blink_threshold && blink_time > blink_interval / 2;
 
-    if (percent && !blink) {
+    if ((percent && !blink)
+        || (g_Config.healthbar_showing_mode == T1M_BSM_DEFAULT)) {
```

Comment:
Can you describe the rationale of this change?

Reply:
In the PS1 OG game, the bars don’t flash when they are low. This makes it so they don't flash if they are low.

Reply:
! That's the thing I needed :)

Reply:
I just checked. HP bar does flash if health is low, but it does not stay up on the screen.

Reply:
Let's restore this behavior then.